package.json

```diff
@@ -74,14 +74,14 @@
     "test": "npm run test-workspaces",
     "test-update": "npm-run-all --sequential clean lint build test",
     "test-workspaces": "lerna run --no-bail test",
-    "prewatch:storybook": "lerna run build:style-dictionary",
+    "prewatch:storybook": "lerna run build",
     "update-patch": "npm-check-updates --configFileName .ncurc.patch.js --deep --dep dev,prod --target patch --upgrade && pnpm install",
     "update-minor": "npm-check-updates --configFileName .ncurc.minor.js --deep --dep dev,prod --target minor --upgrade && pnpm install",
     "update-major": "npm-check-updates --configFileName .ncurc.major.js --deep --dep dev,prod --target latest --upgrade && pnpm install",
     "watch:css": "npm run --workspace packages/css build:watch",
     "watch:react": "npm run --workspace packages/react build:watch",
     "watch:storybook-docs": "npx wait-on http://localhost:6007 && npm run --workspace storybook/storybook-docs start",
     "watch:storybook-react": "npm run --workspace storybook/storybook-react start",
-    "watch:style-dictionary": "npm run --workspace proprietary/tokens watch:style-dictionary"
+    "watch:style-dictionary": "npm run --workspace proprietary/tokens watch"
```

Comment:
```suggestion
    "watch": "npm run --workspace proprietary/tokens watch"
```

Reply:
I'm not sure I understand this suggestion. We're in the root package.json here, where there are `watch` scripts for several packages and both storybooks. All these `watch` scripts together are run with the `"start": "npm-run-all --parallel watch:**"` command.

The script to watch the tokens shouldn't just be called `watch`, I think. We could call it `watch:tokens` or something to make it more clear, though?

Reply:
Yes, `watch:tokens` seems consistent with `:css` and `:react`.